### PR TITLE
Documentation: Added documentation for cross building for Linux Emulator

### PR DESCRIPTION
**Pull request by prajwal-aithal**
Adding documentation for building corefx for Linux ARM Emulator

Signed-off-by: Prajwal A N <an.prajwal@samsung.com>

### CODE REVIEW

**Comment by prajwal-aithal** [comment]
@stephentoub @leemgs Could you please review and merge?

File: Documentation/building/cross-building.md [hunk 1]
```diff
@@ -91,4 +91,22 @@ The output is at bin/<BuildOS>.AnyCPU.Debug.
     drwx------  33 lgs lgs  4096  3  6 21:33 Windows_NT.AnyCPU.Debug
     drwx------  33 lgs lgs  4096  3  6 19:36 Windows_NT.AnyCPU.Release
 
+Building corefx for Linux ARM Emulator
+=======================================
 
+It is possible to build corefx binaries (native and managed) for the Linux ARM Emulator (latest version provided here: [#3805](https://github.com/dotnet/coreclr/issues/3805)).
+The `scripts/arm32_ci_script.sh` script does this.
```
Comment (leemgs) [inline Documentation/building/cross-building.md]: Please, write the location of the "arm32_ci_script.sh" script file in detail for newbies.

Comment (prajwal-aithal) [inline Documentation/building/cross-building.md]: What do you mean? Linking to that file on github?

**Comment by leemgs** [comment]
\CC: @sjsinju

File: Documentation/building/cross-building.md (hunk 1 above)
Comment (leemgs) [inline Documentation/building/cross-building.md]: I mean that the newbies can not know where  "arm32_ci_script.sh" file is located in mounted emulator rootFS.

Comment (prajwal-aithal) [inline Documentation/building/cross-building.md]: This script is in the corefx source.

Comment (leemgs) [inline Documentation/building/cross-building.md]: I thought that the script was also located in /dotnet/setenv of the mounted emulator rootfs. You may keep the existing statement without any modification.

**Comment by prajwal-aithal** [comment]
@leemgs I have improved the documentation. Please take a look at the updated PR?

File: Documentation/building/cross-building.md [hunk 2]
```diff
@@ -91,4 +91,24 @@ The output is at bin/<BuildOS>.AnyCPU.Debug.
...
+
+The following instructions assume that:
+* You have setup the extracted emulator at `/opt/linux-arm-emulator` (such that `/opt/linux-arm-emulator/platform/rootfs-t30.ext4` exists)
```
Comment (stephentoub) [inline Documentation/building/cross-building.md]: Nit: setup => set up

File: Documentation/building/cross-building.md [hunk 3]
```diff
@@ -91,4 +91,24 @@ The output is at bin/<BuildOS>.AnyCPU.Debug.
...
+* The mount path for the emulator rootfs is `/opt/linux-arm-emulator-root` (change this path if you have a working directory at this path).
```
Comment (stephentoub) [inline Documentation/building/cross-building.md]: Does this assume a particular distro/version of Linux?

**Comment by stephentoub** [comment]
Couple nits, otherwise LGTM.

**Comment by leemgs** [comment]
LGTM
Acked-By: Geunsik Lim

File: Documentation/building/cross-building.md (hunk 3 above)
Comment (prajwal-aithal) [inline Documentation/building/cross-building.md]: The script has been tested on Ubuntu. But AFAIK the script should run in any distro of Linux.

**Comment by prajwal-aithal** [comment]
@stephentoub Nits resolved. Could you please review and merge?

**Comment by stephentoub** [comment]
Thanks.

**Pull request merged**